### PR TITLE
Git analysis improvements

### PR DESCRIPTION
**Pull request by gingerwizard**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Changes based on feedback inc. play links from @alexey-milovidov 

@DanRoscigno

### CODE REVIEW

File: docs/en/getting-started/example-datasets/github.md [hunk 1]
```diff
@@ -1730,10 +1816,12 @@ There are a few ways we can address this question. Focusing on the code to test
 
 Note we limit to users with more than 20 changes to focus on regular committers and avoid a bias to one-off contributions.
 
+[play](https://play.clickhouse.com/play?user=play#U0VMRUNUCiAgICBhdXRob3IsCiAgICBjb3VudElmKChmaWxlX2V4dGVuc2lvbiBOT1QgSU4gKCdoJywgJ2NwcCcsICdzaCcsICdweScsICdleHBlY3QnKSkgQU5EIChwYXRoIExJS0UgJyV0ZXN0cyUnKSkgQVMgdGVzdCwKICAgIGNvdW50SWYoKGZpbGVfZXh0ZW5zaW9uIElOICgnaCcsICdjcHAnLCAnc3FsJykpIEFORCAoTk9UIChwYXRoIExJS0UgJyV0ZXN0cyUnKSkpIEFTIGNvZGUsCiAgICBjb2RlIC8gKGNvZGUgKyB0ZXN0KSBBUyByYXRpb19jb2RlCkZST00gZ2l0X2NsaWNraG91c2UuZmlsZV9jaGFuZ2VzCkdST1VQIEJZIGF1dGhvcgpIQVZJTkcgY29kZSA+IDIwCk9SREVSIEJZIGNvZGUgREVTQwpMSU1JVCAyMA==)
+
 ```sql
 SELECT
     author,
-    countIf((file_extension NOT IN ('h', 'cpp')) AND (path LIKE '%tests%')) AS test,
+    countIf((file_extension NOT IN ('h', 'cpp', 'sh', 'py', 'expect')) AND (path LIKE '%tests%')) AS test,
```
Comment (azat) [inline docs/en/getting-started/example-datasets/github.md:1824]: Hm, but there are lots of `.sh`/`.py`/`.expect` tests

Comment (gingerwizard) [reply docs/en/getting-started/example-datasets/github.md]: nice spot thanks, fixed.

File: docs/en/getting-started/example-datasets/github.md [hunk 2]
```diff
@@ -1730,10 +1816,12 @@ There are a few ways we can address this question. Focusing on the code to test
 
 Note we limit to users with more than 20 changes to focus on regular committers and avoid a bias to one-off contributions.
 
+[play](https://play.clickhouse.com/play?user=play#U0VMRUNUCiAgICBhdXRob3IsCiAgICBjb3VudElmKChmaWxlX2V4dGVuc2lvbiBJTiAoJ2gnLCAnY3BwJywgJ3NoJywgJ3B5JywgJ2V4cGVjdCcpKSBBTkQgKHBhdGggTElLRSAnJXRlc3RzJScpKSBBUyB0ZXN0LAogICAgY291bnRJZigoZmlsZV9leHRlbnNpb24gSU4gKCdoJywgJ2NwcCcsICdzcWwnKSkgQU5EIChOT1QgKHBhdGggTElLRSAnJXRlc3RzJScpKSkgQVMgY29kZSwKICAgIGNvZGUgLyAoY29kZSArIHRlc3QpIEFTIHJhdGlvX2NvZGUKRlJPTSBnaXRfY2xpY2tob3VzZS5maWxlX2NoYW5nZXMKR1JPVVAgQlkgYXV0aG9yCkhBVklORyBjb2RlID4gMjAKT1JERVIgQlkgY29kZSBERVNDCkxJTUlUIDIw)
+
 ```sql
 SELECT
     author,
-    countIf((file_extension NOT IN ('h', 'cpp')) AND (path LIKE '%tests%')) AS test,
+    countIf((file_extension IN ('h', 'cpp', 'sh', 'py', 'expect')) AND (path LIKE '%tests%')) AS test,
```
Comment (azat) [inline docs/en/getting-started/example-datasets/github.md:1824]: I guess you want something like this?

```suggestion
    countIf((file_extension IN ('sql', 'sh', 'py', 'expect')) AND (path LIKE '%tests%')) AS test,
```

Comment (gingerwizard) [reply docs/en/getting-started/example-datasets/github.md:1824]: yeh i changed, great spot ta

Comment (azat) [reply docs/en/getting-started/example-datasets/github.md:1876]: Seems that this had been forgotten?

Comment (gingerwizard) [reply docs/en/getting-started/example-datasets/github.md:1876]: sorry @azat maybe im missing something but current commit LGTM..it reads

```sql
SELECT
    author,
    countIf((file_extension IN ('h', 'cpp', 'sh', 'py', 'expect')) AND (path LIKE '%tests%')) AS test,
    countIf((file_extension IN ('h', 'cpp', 'sql')) AND (NOT (path LIKE '%tests%'))) AS code,
    code / (code + test) AS ratio_code
FROM git.file_changes
GROUP BY author
HAVING code > 20
ORDER BY code DESC
LIMIT 20

```

Comment (azat) [reply docs/en/getting-started/example-datasets/github.md:1876]: But there are tests in `sql` (at least), so I guess something like this should be used:

```sql
    countIf((file_extension IN ('h', 'cpp', 'sql', 'sh', 'py', 'expect')) AND (path LIKE '%tests%')) AS test,
```

Comment (gingerwizard) [reply docs/en/getting-started/example-datasets/github.md]: fixed thanks

**Pull request merged**